plugin.rb

```diff
@@ -609,7 +609,7 @@ def self.alloc_id
 
     def self.from_hash(h)
       query = DataExplorer::Query.new
-      [:name, :description, :sql].each do |sym|
+      [:name, :description, :sql, :created_by, :created_at, :username].each do |sym|
```

Comment:
No need to add new fields here since it's not supplied by client.

Reply:
I thought so too, but whenever I remove these fields from `from_hash` and `to_hash`, the values don't show up anymore even after creating a new query. That's why I left them there.

I just tried this again and removing the fields seems to break the table values.

Reply:
I see that `from_hash` is being used to fetch all queries and pass it on to client, so the fields are indeed needed here. Ignore my previous comment.